plugin/storage/es/esCleaner.py

```diff
@@ -58,17 +62,29 @@ def main():
 
 
 def filter_main_indices(ilo, prefix):
-    ilo.filter_by_regex(kind='prefix', value=prefix + "jaeger")
+    ilo.filter_by_regex(kind='regex', value=prefix + "jaeger-(span|service|dependencies)-\d{4}-\d{2}-\d{2}")
     empty_list(ilo, "No indices to delete")
     # This excludes archive index as we use source='name'
     # source `creation_date` would include archive index
     ilo.filter_by_age(source='name', direction='older', timestring='%Y-%m-%d', unit='days', unit_count=int(sys.argv[1]))
 
 
-def filter_archive_indices(ilo, prefix):
-    # Remove only archive indices when aliases are used
+def filter_main_indices_rollover(ilo, prefix):
+    ilo.filter_by_regex(kind='regex', value=prefix + "jaeger-(span|service)-\d{6}")
+    empty_list(ilo, "No indices to delete")
+    # do not remove active write indices
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-span-write'], exclude=True)
```

Comment:
If filtering by the regex now, are these two `aliases` filters required?

Reply:
See the comment above. We need to exclude current active write indices.